### PR TITLE
AWS: Create a S3 Storage Lens configuration

### PR DESCRIPTION
**Pull request by ameukam**
Create a S3 Storage Lens config (dashboard) that will help have a central overview of the S3 buckets at the org level.

### CODE REVIEW

**Comment by ameukam** [comment]
cc @sftim @deobieta

File: infra/aws/terraform/audit-account/s3_storage_lens.tf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      }
+    }
+  }
+
+  tags = merge({
+    env       = "Audit",
+    component = "Security",
+    service   = "S3 Storage Lens"
+  }, var.tags)
+}
```
Comment (sftim) [inline infra/aws/terraform/audit-account/s3_storage_lens.tf:57]: Should we define at least one IAM role that is able to access this dashboad?

(I would).

**Comment by ameukam** [comment]
Ref: https://github.com/kubernetes/k8s.io/issues/4654

File: infra/aws/terraform/audit-account/s3_storage_lens.tf (hunk 1 above)
Comment (ameukam) [reply infra/aws/terraform/audit-account/s3_storage_lens.tf:57]: yeah. Wanted to do this as a follow-up and maybe wait to get the final approach on how we want to do federated access (See #4746).

File: infra/aws/terraform/audit-account/s3_storage_lens.tf [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    aws_org {
+      arn = data.aws_organizations_organization.current.arn
+    }
+
+    data_export {
+      cloud_watch_metrics {
+        enabled = true
+      }
+    }
```
Comment (sftim) [inline infra/aws/terraform/audit-account/s3_storage_lens.tf:49]: (optional)
We could add a comment that this export is once per day.

Comment (ameukam) [reply infra/aws/terraform/audit-account/s3_storage_lens.tf]: Done.

**Review by upodroid — APPROVED** [review]
/lgtm

**Pull request merged**